src/pymorize/units.py

```diff
@@ -110,15 +98,24 @@ def handle_unit_conversion(da: xr.DataArray, rule: Rule) -> xr.DataArray:
     model_unit = rule.get("model_unit")
     from_unit = da.attrs.get("units")
     if model_unit is not None:
-        logger.debug(
+        logger.info(
             f"using user defined unit ({model_unit}) instead of ({from_unit}) from DataArray "
         )
         from_unit = model_unit
     handle_chemicals(from_unit)
     handle_chemicals(to_unit)
     new_da = da.pint.quantify(from_unit)
-    logger.debug(f"Converting units: {from_unit} -> {to_unit}")
-    new_da = new_da.pint.to(to_unit).pint.dequantify()
+    dimless = rule.get("dimensionless_unit_mappings", {})
+    cmor_variable = rule.get("cmor_variable")
+    if cmor_variable in dimless:
+        _to_unit = dimless[cmor_variable][to_unit]
+    else:
```

Comment:
Somewhere here, there should be a dimensionless, or "weird-unit" checker, so that it raises and error if a variable won't be able to be transformed by `pint` and suggesting the user/developer to include a mapping of units in the dimensionless_unit_mappings